libnetwork/portallocator/portallocator.go

```diff
@@ -96,9 +96,10 @@ func newInstance() *PortAllocator {
 		start, end = defaultPortRangeStart, defaultPortRangeEnd
 	}
 	return &PortAllocator{
-		ipMap: ipMapping{},
-		Begin: start,
-		End:   end,
+		ipMap:     ipMapping{},
+		defaultIP: net.IPv4zero,
```

Comment:
I'm not really tied to this; I initially kept it as a package variable, but decided to move it internal to the PortAllocator, mostly because that's already a singleton and to make it more clear it's internal to that instance, but also (slightly) considering if this would be something configurable (i.e. when instancing the allocator).